internal/function_window_option.go

```diff
@@ -406,32 +406,33 @@ func (s *WindowFuncAggregatedStatus) Done(cb func([]Value, int, int) error) erro
 	sortedValues := make([]*WindowOrderedValue, len(values))
 	copy(sortedValues, values)
 	if len(sortedValues) != 0 {
-		for orderBy := 0; orderBy < len(sortedValues[0].OrderBy); orderBy++ {
-			isAsc := sortedValues[0].OrderBy[orderBy].IsAsc
-			if isAsc {
-				sort.Slice(sortedValues, func(i, j int) bool {
-					if sortedValues[i].OrderBy[orderBy].Value == nil {
-						return true
-					}
-					if sortedValues[j].OrderBy[orderBy].Value == nil {
-						return false
-					}
-					cond, _ := sortedValues[i].OrderBy[orderBy].Value.LT(sortedValues[j].OrderBy[orderBy].Value)
+		sort.Slice(sortedValues, func(i, j int) bool {
```

Comment:
We should share this implementation with `function_aggregate.go`

Reply:
OK, It's good. thank you.